test/Microsoft.DocAsCode.Build.Common.Tests/ModelAttributeHandlerTest.cs

```diff
@@ -263,11 +263,34 @@ public void TesteModelWithIListMarkdownContentAttributeShouldSucceed()
 
         #endregion
 
+        #region HrefAttribute
+
+        [Fact]
+        public void TestSimpleModelWithHrefAttributeShouldSucceed()
+        {
+            var model = new SimpleModel
+            {
+                Href = "linkTarget"
```

Comment:
>Href [](start = 16, length = 4)

so before processing, it is the href relative to current model, and after processing, it is the href relative to root?